### PR TITLE
Add QSG-3

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/fragments/fb-claim-mapping.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    Claim mapping Dialect**.
+    
+5.  Click **Add Claim Mapping.** The following screen will be
+    displayed.  
+    ![add-claim-mapping-for-facebook](../../../assets/img/fragments/add-claim-mapping-for-facebook.png)
+    
+6.  Now add custom claim mappings as follows.  
+    ![custom-claim-mapping](../../../assets/img/fragments/custom-claim-mapping.png)
+    
+7.  You can retrieve all the public information of the user and the
```
Comment (dewniMW) [inline en/docs/fragments/fb-claim-mapping.md:30]: this is part of step 6 right? shall we make this a note?

File: en/docs/fragments/fb-review.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
-!!! info "About accessing the app"
-
+!!!	info "About accessing the app"
 	The app is not available to general public yet. To make to app available
```
Comment (dewniMW) [inline en/docs/fragments/fb-review.md:2]:
```suggestion
	The app is not available to the general public yet. To make the app available
```

File: en/docs/fragments/idp-facebook.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    </tr>
+    <tr class="even">
+    <td>User Information Fields</td>
+    <td>These are the claims related to the user account on Facebook. WSO2 Identity Server requests these fields from Facebook when a user is authenticated with Facebook through the IS. See <a href="https://developers.facebook.com/docs/facebook-login/permissions#reference-public_profile">public_profile permission</a> for more information about these fields.</td>
+    <td>id,name,gender,email,first_name,last_name,age_range,link</td>
+    </tr>
+    <tr class="odd">
+    <td>Callback Url</td>
+    <td>This is the URL to which the browser should be redirected after the authentication is successful. This should be the commonauth endpoint of Identity server.</td>
+    <td><a href="https://localhost:9443/commonauth">https://localhost:9443/commonauth</a></td>
```
Comment (dewniMW) [inline en/docs/fragments/idp-facebook.md:54]: https://<host>:<port>/commonauth

File: en/docs/quick-starts/client-credentials-playground.md [hunk 4]
```diff
@@ -18,9 +18,9 @@ This page guides you through using a **sample Playground application** to try ou
 
 	- **Client Secret:** The Client Secret recieved when registering the service provider.
 
-	- **Access Token Endpoint:** https://localhost:9443/oauth2/token
+	- **Access Token Endpoint:** `https://localhost:9443/oauth2/token`
```
Comment (dewniMW) [inline en/docs/quick-starts/client-credentials-playground.md:21]: https://<IS_HOST>:<IS_PORT>/oauth2/token

File: en/docs/quick-starts/client-credentials-playground.md [hunk 5]
```diff
@@ -47,14 +47,14 @@ This page guides you through using a **sample Playground application** to try ou
 	grant_type=client_credentials
 	``` 
 
-3. Log in with user credentials (e.g., admin/admin). At this point, the application receives the access token. 
+3. Log in with user credentials (e.g., `admin`/`admin`). At this point, the application receives the access token. 
 
-4. Enter the **Introspection Endpoint** (i.e, https://localhost:9443/oauth2/introspect) and click **Get TokenInfo** to get the token   information. 
-
-	For more information, see [OAuth Token Introspection](insertlink).
+4. Enter the **Introspection Endpoint** (i.e, `https://localhost:9443/oauth2/introspect`) and click **Get TokenInfo** to get the token   information.
```
Comment (dewniMW) [inline en/docs/quick-starts/client-credentials-playground.md:52]: `https://<IS_HOST>:<IS_PORT>/oauth2/introspect`

File: en/docs/quick-starts/password-playground.md [hunk 6]
```diff
@@ -55,12 +55,10 @@ This page guides you through using a **sample Playground application** to try ou
 
 3. Log in with user credentials (e.g., admin/admin). At this point, the application receives the access token. 
 
-4. Enter the **Introspection Endpoint** (i.e, https://localhost:9443/oauth2/introspect) and click **Get TokenInfo** to get the token   information. 
-
-	For more information, see [OAuth Token Introspection](insertlink).
+4. Enter the **Introspection Endpoint** (i.e, `https://localhost:9443/oauth2/introspect`) and click **Get TokenInfo** to get the token information.
```
Comment (dewniMW) [inline en/docs/quick-starts/password-playground.md:58]: shall we make it `<IS_HOST>:<PORT>`? check all applicable places

File: en/docs/quick-starts/twitter-as-federated-authenticator-sample.md [hunk 7]
```diff
@@ -57,7 +57,60 @@ If you have your own application, click the button below.
...
+    <td><div class="row">
+    <code>                771tqnkpcbRyTBSCRQvVud1x8j1uQlCDpNZo3hRG0s4cEtsFky               </code>
+    </div></td>
+    </tr>
+    <tr class="odd">
+    <td>Callback URL</td>
+    <td><p>This is the Callback URL you entered at the Twitter application registration. This is the URL to which the browser should be redirected after the authentication is successful.</p>
+    <p>URL format: <code>                https://&lt;host-name&gt;:&lt;port&gt;/acs               </code></p>
+    <p>The acs indicates the Assertion Consumer URL of the WSO2 Identity Server endpoint that accepts the responses sent by Twitter.</p></td>
+    <td><code>                               https://wso2.com:9443/commonauth                             </code></td>
```
Comment (dewniMW) [inline en/docs/quick-starts/twitter-as-federated-authenticator-sample.md:108]: <IS_HOST>:<PORT>

File: en/docs/fragments/fb-claim-mapping.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
+All the basic information of a user/application is stored in the form of
+claims. But for the same information, different Identity Providers(IDP)
```
Comment (dewniMW) [inline en/docs/fragments/fb-claim-mapping.md:2]:
```suggestion
claims. However for the same information, different Identity Providers(IDP)
```

File: en/docs/quick-starts/auth-code-playground.md [hunk 9]
```diff
@@ -20,21 +20,21 @@ This page guides you through using a **sample Playground application** to try ou
 
     - **Client ID:** The OAuth Client Key recieved when registering the service provider.
 
-    - **Callback URL:** http://wso2is.local:8080/playground2/oauth2client
+    - **Callback URL:** `http://wso2is.local:8080/playground2/oauth2client`
 
-    - **Authorize Endpoint:** https://localhost:9443/oauth2/authorize
+    - **Authorize Endpoint:** `https://localhost:9443/oauth2/authorize`
```
Comment (dewniMW) [inline en/docs/quick-starts/auth-code-playground.md:25]: `<IS_HOST>:<PORT>`

File: en/docs/quick-starts/auth-code-playground.md [hunk 10]
```diff
@@ -61,21 +61,19 @@ This page guides you through using a **sample Playground application** to try ou
 
 4. Provide the requested consent and enter the following details on the screen that appears. 
 
-    - **Callback URL:** http://wso2is.local:8080/playground2/oauth2client
+    - **Callback URL:** `http://wso2is.local:8080/playground2/oauth2client`
 
-    - **Access Token Endpoint:** https://localhost:9443/oauth2/token 
+    - **Access Token Endpoint:** `https://localhost:9443/oauth2/token`
```
Comment (dewniMW) [inline en/docs/quick-starts/auth-code-playground.md:66]: `<IS_HOST>:<IS_PORT>`

File: en/docs/quick-starts/auth-code-playground.md [hunk 11]
```diff
@@ -61,21 +61,19 @@ This page guides you through using a **sample Playground application** to try ou
...
     - **Client Secret:** The client secret received when registering the service provider.
 
 	- **PKCE Verifier:** This will be populated using the value generated in step 1 only if you are using PKCE. 
 
 5. Click **Get Access Token**. At this point, the application receives the access token. 
 
-6. Enter the **Introspection Endpoint** (i.e, https://localhost:9443/oauth2/introspect) and click **Get TokenInfo** to get the token information. 
-
-	For more information, see [OAuth Token Introspection](insertlink).
+6. Enter the **Introspection Endpoint** (i.e, `https://localhost:9443/oauth2/introspect`) and click **Get TokenInfo** to get the token information.
```
Comment (dewniMW) [inline en/docs/quick-starts/auth-code-playground.md:74]: `<IS_HOST>:<IS_PORT>`

File: en/docs/quick-starts/implicit-playground.md [hunk 12]
```diff
@@ -16,11 +16,11 @@ This page guides you through using a **sample Playground application** to try ou
     
     - **Client ID:** The OAuth Client Key recieved when registering the service provider.
     
-    - **Callback URL:** http://wso2is.local:8080/playground2/oauth2client
+    - **Callback URL:** `http://wso2is.local:8080/playground2/oauth2client`
 
-	- **Authorize Endpoint:** https://localhost:9443/oauth2/authorize
+	- **Authorize Endpoint:** `https://localhost:9443/oauth2/authorize`
```
Comment (dewniMW) [inline en/docs/quick-starts/implicit-playground.md:21]: `<IS_HOST>:<IS_PORT>`

File: en/docs/quick-starts/fb-as-federated-authenticator-sample.md [hunk 13]
```diff
@@ -18,29 +18,29 @@ Follow the steps given below to configure WSO2 Identity Server to authenticate u
...
 5. Complete the security check and click **Submit**. 
 
 6. Click **Set up** under Facebook Login.
 
-	![setup-fb](../../assets/img/samples/facebook-login.png)
+	![setup-fb](../../../assets/img/samples/facebook-login.png)
 
 7. Select **Website** as the platform for the app used in this sample.
 
 8. Enter `https://localhost:9443/` as the **Site URL** and click **Save**.
```
Comment (dewniMW) [inline en/docs/quick-starts/fb-as-federated-authenticator-sample.md:37]: `<IS_HOST>:<IS_PORT>`

File: en/docs/quick-starts/oidc-token-encryption-sample.md [hunk 14]
```diff
@@ -116,9 +116,9 @@ You have successfully set up token encryption and the sample Playground applicat
 
     -   **Scope:** openid 
 
-    -   **Callback URL:** <http://wso2is.local:8080/playground2/oauth2client>
+    -   **Callback URL:** `http://wso2is.local:8080/playground2/oauth2client`
 
-    -   **Authorize Endpoint:** <https://localhost:9443/oauth2/authorize>
+    -   **Authorize Endpoint:** `https://localhost:9443/oauth2/authorize`
```
Comment (dewniMW) [inline en/docs/quick-starts/oidc-token-encryption-sample.md:121]: `<IS_HOST>:<IS_PORT>` check all applicable places

File: en/docs/quick-starts/auth-code-playground.md (hunk 10 above)
Comment (gomathyk) [reply en/docs/quick-starts/auth-code-playground.md:66]: Leaving it with the values since it's a sample.

**Review by dewniMW — APPROVED** [review]

**Pull request merged**